ui/components/app/account-menu/account-menu.component.js

```diff
@@ -205,7 +206,9 @@ export default class AccountMenu extends Component {
               <div className="account-menu__check-mark-icon" />
             ) : null}
           </div>
-          <Identicon address={identity.address} diameter={24} />
+          {/* <Identicon address={identity.address} diameter={24} /> */}
```

Comment:
- I think instead of using fixed image, we should use identicon for now
- We will discuss whether we should use our own identicon in the future

Reply:
fixed in 36b42cb71c5b58b72578384b41968443627becfe